zephyr/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ if(CONFIG_INTEL_HAL)
 
   zephyr_library_sources(${BSP_SEDI_SRC}/drivers/rtc/sedi_rtc.c)
   zephyr_library_sources_ifdef(CONFIG_I2C_SEDI ${BSP_SEDI_SRC}/drivers/i2c/sedi_i2c_dw_apb_200a.c)
+  zephyr_library_sources_ifdef(CONFIG_PM ${BSP_SEDI_SRC}/drivers/pm/ish_pm.c)
```

Comment:
put to the end of this file, wrap with ISH Kconfig.

Reply:
done.